SteamList/Controllers/FavoritesViewController.swift

```diff
@@ -7,23 +7,33 @@
 
 import UIKit
 
-final class FavoritesViewController: UIViewController {
+final class FavoritesViewController: UIViewController, UISearchResultsUpdating {
```

Comment:
UISearchResultsUpdating could be written in an empty extension of this class for clearance of style

Reply:
done